src/models/consent.ts

```diff
@@ -40,18 +40,28 @@ export enum ConsentStatus {
    * Waiting for the user to confirm payee information and provide more
    * details about the transaction.
    */
-  PENDING_PAYEE_CONFIRMATION = 'PENDING_PAYEE_CONFIRMATION',
+  PENDING_PARTY_CONFIRMATION = 'PENDING_PARTY_CONFIRMATION',
 
   /**
    * Waiting for the user to authorize the consent.
    */
-  AUTHORIZATION_REQUIRED = 'AUTHORIZATION_REQUIRED',
+  AUTHENTICATION_REQUIRED = 'AUTHENTICATION_REQUIRED',
```

Comment:
there is a difference between Autentication and Authorization, comment is telling about AUTHORIZATION of the consent so I would keep it as it is

Reply:
Same with my other comment, I think it's not an authorization since this is performed when establishing consent, not for authorizing a transfer. In this phase, the DFSP will ask user to perform authentication on their website or perhaps via OTP. Hence, I think the word "Authentication" better suits this case rather than "Authorization".